src/devices/video/topcat.h

```diff
@@ -21,7 +24,8 @@ class topcat_device : public device_t
 
 	void set_fb_width(int _pixels) { m_fb_width = _pixels; }
 	void set_fb_height(int _pixels) { m_fb_height = _pixels; }
-	void set_planes(int _planes) { m_planes = _planes; }
+	void set_planemask(int _mask) { m_plane_mask = _mask; }
+	void set_vram(std::vector<uint8_t> *vram) { m_vram = vram; }
```

Comment:
This is better handled by exposing a memory space.

Reply:
Do you have a good example how to use a memory space here? I'm looking for some working device/driver so i can unterstand the concept.

Reply:
See e.g. pull request #3515 that converts the R5C68 to use expose a memory space.